api/client/search.go

```diff
@@ -60,7 +71,14 @@ func (cli *DockerCli) CmdSearch(args ...string) error {
 	w := tabwriter.NewWriter(cli.out, 10, 1, 3, ' ', 0)
 	fmt.Fprintf(w, "NAME\tDESCRIPTION\tSTARS\tOFFICIAL\tAUTOMATED\n")
 	for _, res := range results {
-		if (*automated && !res.IsAutomated) || (int(*stars) > res.StarCount) {
+		if filterOptions["is-automated"] == "true" && !res.IsAutomated {
```

Comment:
Should we also support the opposite; `--filter is-automated=false` to only show non-automated results?

(same for `is-official`?)